content/pages/hem.md

```diff
@@ -4,10 +4,12 @@ date: 2023-11-18
 author: Anders Nord
 type: hem
 status: published
-data: poster2.jpeg
+data:
 ---
 
-## Vi vill ha en odlingshub i Majorna?
+<img src="data/poster2.jpeg" width=100% alt="Poster.">
```

Comment:
```suggestion
<img src="/hem/data/poster2.jpeg" width=100% alt="Poster.">
```

Should work for now. Data for pages is still placed under a folder of that page, even though "hem" is the root index.